src/components/LoginOptions/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Types from 'types';
+import PropTypes from 'prop-types';
+import Literal from 'components/Literal';
+import {getLoginUrl} from 'components/utils';
+
+import LoginOptionsDisplay from './LoginOptionsDisplay';
+
+const LoginOptions = ({form, onFormStart}) => {
+  const loginAsYourselfOptions = [];
+  const loginAsGemachtigdeOptions = [];
+
+  form.loginOptions.forEach(option => {
+    let readyOption = {...option};
+    readyOption.url = getLoginUrl(option);
+    readyOption.label = (
+      <FormattedMessage
+        description="Login button label"
+        defaultMessage="Login with {provider}"
+        values={{provider: option.label}}
+      />
+    );
+
+    if (readyOption.isForGemachtigde) {
+      loginAsGemachtigdeOptions.push(readyOption);
+    } else {
+      loginAsYourselfOptions.push(readyOption);
+    }
+  });
+
+  if (!form.loginRequired) {
+    loginAsYourselfOptions.push({
+      identifier: 'anonymous',
+      label: <Literal name="beginText" />,
+      url: '#',
+      onClick: onFormStart,
+    });
```

Comment:
maybe we should make this `onFormStart` thing more semantic:

1. if login is not required, render the form start page between `form` tags, otherwise use a `div`
2. render the anonymous login option/button as an actual `type="submit"` button, when it's clicked, it submits the outer `form`
3. hook up the outer `form` with `onSubmit={onFormStart}`. This decouples the form start event from the actual buttons/links. Because the login buttons are links to the backend, they will not result in a form start submit and instead invoke the form start via the URL query parameter.

This approach allows us to not have to deal with `onClick` at all in login button components.

Reply:
I changed it, although now there is more logic in the `LoginButton` component to figure out the URLs/type of component/... :thinking: But I like not having the `onClick`